### PR TITLE
Creating Ghost Secondary variant

### PR DESCRIPTION
**Pull request by Duncid**
## Description

New button variant (ghost-secondary)

### CODE REVIEW

File: sparkle/src/components/IconButton.tsx [hunk 1]
```diff
@@ -42,6 +42,11 @@ const styleVariants: Record<ButtonVariantType, string> = {
     "hover:s-text-slate-100 hover:s-text-slate-100" +
     "active:s-text-slate-200 active:s-text-slate-200" +
     "s-text-white/50 s-text-white/50",
+  "ghost-secondary":
+    "s-text-white s-text-white" +
```
Comment (JulesBelveze) [inline sparkle/src/components/IconButton.tsx:46]:
```suggestion
    "s-text-white" +
```

File: sparkle/src/components/IconButton.tsx [hunk 2]
```diff
@@ -42,6 +42,11 @@ const styleVariants: Record<ButtonVariantType, string> = {
...
+    "hover:s-text-slate-100 hover:s-text-slate-100" +
+    "active:s-text-slate-200 active:s-text-slate-200" +
+    "s-text-white/50 s-text-white/50",
```
Comment (JulesBelveze) [inline sparkle/src/components/IconButton.tsx:49]: Okay I think we have a bunch of duplicates here 😅 Would you mind getting rid of it?

Comment (Duncid) [reply sparkle/src/components/IconButton.tsx]: I'd love to.
I don't know why IconButton's variants are mapped to Button ones, it makes no sense.

I don't remember doing it so I did not revert.

Comment (Duncid) [reply sparkle/src/components/IconButton.tsx]: Since we want to retire them, I figured it did not require addressing.

**Review by JulesBelveze — APPROVED** [review]

**Pull request merged**